src/wolff.appup.src

```diff
@@ -1,8 +1,9 @@
 %% -*-: erlang -*-
-{"1.5.5",
+{"1.5.6",
   [
-    {<<"1\\.5\\.[2-4]">>,
-     [ {load_module, wolff_producer, brutal_purge, soft_purge, []}
+    {<<"1\\.5\\.[2-5]">>,
+     [ {add_application, lc}
```

Comment:
We don't need this `add_application`, `systools` will detect new app and load it in relup.